README.md

```diff
@@ -145,133 +145,53 @@ You can send metrics, histograms, or trace data from your application to the Wav
+  * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
 
-> **Note**: See [Advanced Proxy Configuration and Installation](https://docs.wavefront.com/proxies_configuring.html) for details.
+### Initialize the WavefrontClient
 
-```java
-// Create the builder with the proxy hostname or address
-WavefrontProxyClient.Builder wfProxyClientBuilder = new WavefrontProxyClient.Builder(proxyHostName);
-
-// Set the proxy port to send metrics to. Default: 2878
-wfProxyClientBuilder.metricsPort(2878);
-
-// Set a proxy port to send histograms to.  Recommended: 2878
-wfProxyClientBuilder.distributionPort(2878);
-
-// Set a proxy port to send trace data to. Recommended: 30000
-wfProxyClientBuilder.tracingPort(30_000);
-
-// Optional: Set a custom socketFactory to override the default SocketFactory
-wfProxyClientBuilder.socketFactory(<SocketFactory>);
-
-// Optional: Set a nondefault interval (in seconds) for flushing data from the sender to the proxy. Default: 5 seconds
-wfProxyClientBuilder.flushIntervalSeconds(2);
-
-// Create the WavefrontProxyClient
-WavefrontSender wavefrontSender = wfProxyClientBuilder.build();
-```
-
-> **Note:** When you set up a Wavefront proxy on the specified proxy host, you specify the port it will listen to for each type of data to be sent. The `WavefrontProxyClient` must send data to the same ports that the Wavefront proxy listens to. Consequently, the port-related builder methods must specify the same port numbers as the corresponding proxy configuration properties. See the following table: 
-
-| `WavefrontProxyClient` builder method | Corresponding property in `wavefront.conf` |
-| ----- | -------- |
-| `metricsPort()` | `pushListenerPorts=` |
-| `distributionPort()` | `histogramDistListenerPorts=` |
-| `tracingPort()` | `traceListenerPorts=` |
-
-### Option 2: Sending Data via Direct Ingestion
-
-When sending data via direct ingestion, you need to create a `WavefrontDirectIngestionClient`, and build it with the Wavefront URL and API token to send data directly to Wavefront.
-
->**Prerequisites**
-> * Verify that you have the Direct Data Ingestion permission. For details, see [Examine Groups, Roles, and Permissions](https://docs.wavefront.com/users_account_managing.html#examine-groups-roles-and-permissions).
-> * The URL of your Wavefront instance. This is the URL you connect to when you log in to Wavefront, typically something like `https://<domain>.wavefront.com`.
-> * [Obtain the API token](http://docs.wavefront.com/wavefront_api.html#generating-an-api-token).
-
-#### Initialize the WavefrontDirectIngestionClient
-You initialize a `WavefrontDirectIngestionClient` by building it with the access information you obtained in the Prerequisites section.
+You initialize a `WavefrontClient` by building it with the information you obtained in the Prerequisites section.
```

Comment:
The `WavefrontClient` also now has a builder method (for those that need to pass a builder around vs. a Factory). 

```java
// Proxy based ingestion
   new WavefrontClient.Builder(proxyServerWithPort).
        messageSizeBytes(messageSizeInBytes).
        batchSize(batchSize).
        flushIntervalSeconds(flushIntervalSeconds).
        maxQueueSize(queueSize).
        build(); // Returns a WavefrontClient

// Wavefront Direct Ingestion
   new WavefrontClient.Builder(cluster, token).
        messageSizeBytes(messageSizeInBytes).
        batchSize(batchSize).
        flushIntervalSeconds(flushIntervalSeconds).
        maxQueueSize(queueSize).
        build(); // Returns a WavefrontClient
``` 

We should ensure these get added to the documentation (the Builder was added in a point release after 2.0)

Reply:
@MikeMcMahon : Thank you! Added two new examples with this info. Is this ok?